assets/package.json

```diff
@@ -4,8 +4,12 @@
   "version": "0.0.1",
   "license": "UNLICENSED",
   "private": true,
-  "main": "gulp.js",
+  "main": "assets/gulp.js",
```

Comment:
Right now `package.json` and `gulpfile.js` are in the same folder itself, right?

Reply:
Reverted.